src/index.php

```diff
@@ -38,13 +41,16 @@
 
     // Send announcements
     if (isset($_POST["discord"])) {
-        send_discord_msg($title, $description . "\n\n" . $signature, $url);
+        send_discord_msg($title, $tag, $description . "\n\n" . $signature, $url);
```

Comment:
Tag is implemented as a part of title, thus it could be passed in $title variable without editing actual integration functions.
Something like:
```
// Include tag within title
if(isset($_POST["tag"])) {
    $title = "[{$tag}] {$title}";
}
```